bundles/core/org.eclipse.smarthome.core.test/pom.xml

```diff
@@ -23,22 +23,45 @@
   <build>
     <plugins>
       <plugin>
-        <groupId>org.eclipse.tycho</groupId>
+        <groupId>${tycho-groupid}</groupId>
+        <artifactId>target-platform-configuration</artifactId>
+        <configuration>
+          <environments combine.self="override"></environments>
+          <dependency-resolution>
+            <extraRequirements>
+              <requirement>
+                <type>eclipse-feature</type>
+                <id>org.eclipse.smarthome.feature.test</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>org.eclipse.equinox.ds</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>org.eclipse.equinox.event</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>com.eclipsesource.jaxrs.jersey-min</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>com.eclipsesource.jaxrs.jersey-all</id>
```

Comment:
> And why should we remove the dependency to the test feature?

Tycho-surefire calculates the dependencies automatically. As most tests inherit from OSGiTest it usually automatically adds the bundle to the list of started bundles for a test. The test feature only contains the test bundle.

At the moment all tests are working fine without the explicit reference of the feature or any other bundle. We usually only reference bundles which are needed by indirect dependencies (mainly DS and EventAdmin) and sometimes we need to start some other bundles like the XML bundles or the Core Bundle.